lib/src/runner/configuration/load.dart

```diff
@@ -289,6 +266,32 @@ class _ConfigurationLoader {
         excludeTags: excludeTags);
   }
 
+  Map _loadFoldedStackFrames() {
```

Comment:
Document this. In particular, it's not clear what the semantics of the return value are.